platform-visualization/src/tileManager.js

```diff
@@ -503,10 +547,10 @@ function TileManager() {
         var mesh,
             element = new THREE.LOD(),
             levels = [
-                ['high', 0],
-                ['medium', 1000],
-                ['small', 1800],
-                ['mini', 2300]
+                ['high', 90000],
+                ['medium', 30000],
+                ['small', 10000],
+                ['mini', 0]
```

Comment:
Did you test this? This means that when the Camera is 90k+ units away, the details will be *high*, but it should be *mini*